sopel/coretasks.py

```diff
@@ -162,28 +221,41 @@ def execute_perform(bot, trigger):
 def startup(bot, trigger):
     """Do tasks related to connecting to the network.
 
-    001 RPL_WELCOME is from RFC2812 and is the first message that is sent after
-    the connection has been registered on the network.
+    ``001 RPL_WELCOME`` is from RFC2812 and is the first message that is sent
+    after the connection has been registered on the network.
 
-    251 RPL_LUSERCLIENT is a mandatory message that is sent after client
+    ``251 RPL_LUSERCLIENT`` is a mandatory message that is sent after client
```

Comment:
```suggestion
    ``251 RPL_LUSERCLIENT`` is a mandatory message that is sent after the client
```

I want this, but I am far too lazy to rewrap the entire paragraph in a suggestion while sitting in a laundromat wearing my COVID-appropriate face mask. 😷 (Also I want to get through the rest of the PR.)